app/services/document_publishing_service.rb

```diff
@@ -14,7 +14,7 @@ def publish_draft(document)
   def publish(document, review_state)
     document.update!(publication_state: "sending_to_live", review_state: review_state)
     publish_assets(document.images)
-    GdsApi.publishing_api_v2.publish(document.content_id, nil, locale: document.locale)
+    GdsApi.publishing_api_v2.publish(document.content_id, document.update_type, locale: document.locale)
```

Comment:
We shouldn't actually do this. Sending update type here is deprecated. 

The method arguments on GDS API Adapters are somewhat unfortunate for this.

It has some effect on change note that I can't quite remember: https://github.com/alphagov/publishing-api/blob/45b3b427c8f4330fae45367cd79ade3961ff75ee/app/commands/v2/publish.rb#L36

Reply:
Thanks for spotting - all gone :-).